.github/workflows/pr.yaml

```diff
@@ -54,5 +54,5 @@ jobs:
     with:
       build_type: pull-request
       # Package is pure Python and only ever requires one build.
-      matrix_filter: 'map(select(.ARCH == "amd64" and (.LINUX_VER | test("centos")|not))) | sort_by(.PY_VER | split(".") | map(tonumber)) | [.[-1]]'
-      script: "ci/test_wheel.sh"
\ No newline at end of file
+      matrix_filter: 'map(select((.LINUX_VER | test("centos")|not))) | sort_by((.PY_VER | split(".") | map(tonumber))) | [.[-1] + {ARCH: "amd64"}]'
```

Comment:
The `matrix_filter` for `wheel-build` and `wheel-test` need to be identical. Otherwise, there's a risk of selecting different artifact names (with different Python versions in them), which can lead to the tests failing like this:

> download failed: s3://rapids-downloads/ci/jupyterlab-nvdashboard/pull-request/205/0de97c7/jupyterlab-nvdashboard_wheel_python_jupyterlab-nvdashboard_py310_x86_64.tar.gz to - An error occurred (404) when calling the HeadObject operation: Not Found


([build link](https://github.com/rapidsai/jupyterlab-nvdashboard/actions/runs/10929317845/job/30366838960?pr=205))